code/modules/telesci/bscrystal.dm

```diff
@@ -15,9 +15,16 @@
 	pixel_y = rand(-5, 5)
 
 /obj/item/bluespace_crystal/attack_self(var/mob/user)
-	blink_mob(user)
+	var/turf/T = get_turf(src)
+	var/datum/zLevel/path = map.zLevels[T.z]
```

Comment:
HA, yes, but the original PATHS in the list never get removed, so a zlevels list(/z1, /z2) ends up list(/z1, /z2, /z1 datum, /z2 datum)